packages/@uppy/xhr-upload/src/index.js

```diff
@@ -214,21 +216,21 @@ module.exports = class XHRUpload extends Plugin {
     this.addMetadata(formPost, meta, opts)
 
     files.forEach((file) => {
-      const opts = this.getOptions(file)
+      const options = this.getOptions(file)
 
       const dataWithUpdatedType = setTypeInBlob(file)
 
       if (file.name) {
-        formPost.append(opts.fieldName, dataWithUpdatedType, file.name)
+        formPost.append(options.fieldName, dataWithUpdatedType, file.name)
       } else {
-        formPost.append(opts.fieldName, dataWithUpdatedType)
+        formPost.append(options.fieldName, dataWithUpdatedType)
       }
     })
 
     return formPost
   }
 
-  createBareUpload (file, opts) {
+  static createBareUpload (file) {
     return file.data
   }
```

Comment:
Should we rename this method in 2.0? It's not creating anything. (Should we remove it even?) Can you add a TODO comment please?